usr/lib/greenboot/check/required.d/02_watchdog.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+set -eo pipefail
+
+source_configuration_file() {
+  GREENBOOT_CONFIGURATION_FILE=/etc/greenboot/greenboot.conf
+  if test -f "$GREENBOOT_CONFIGURATION_FILE"; then
+    # shellcheck source=etc/greenboot/greenboot.conf
+    source $GREENBOOT_CONFIGURATION_FILE
+  fi
+}
+
+set_grace_period() {
+    DEFAULT_GRACE_PERIOD=24 # default to 24 hours
+
+    if [ -n "$GREENBOOT_WATCHDOG_GRACE_PERIOD" ]; then
+        GRACE_PERIOD=$GREENBOOT_WATCHDOG_GRACE_PERIOD
+    else
+        GRACE_PERIOD=$DEFAULT_GRACE_PERIOD
+    fi
+}
+
+check_if_there_is_a_watchdog() {
+    if wdctl 2>/dev/null ; then
+      return 0
+    else
+      return 1
+    fi
+}
+
+check_if_current_boot_is_wd_triggered() {
+    if check_if_there_is_a_watchdog ; then
+      WDCTL_OUTPUT=$(wdctl --flags-only --noheadings | grep -c '1$' || true)
+      if [ "$WDCTL_OUTPUT" -gt 0 ]; then
+        # This means the boot was watchdog triggered
```

Comment:
Should this be changed to "This means the boot was _triggered by the watchdog_" instead?